src/test/java/com/fitwsarah/fitwsarah/accountsubdomain/presentationlayer/AccountControllerIntegrationTest.java

```diff
@@ -1,49 +1,90 @@
 
-    private List<AccountResponseModel> accountResponseModelList;
 
     @BeforeEach
     void setUp() {
-        accountResponseModelList = Arrays.asList(account1);
+        AccountIdentifier identifier = new AccountIdentifier();
+        testAccount = new Account("John", "Doe", "d", "d");
+        testAccount.setAccountIdentifier(identifier);
+        Account savedAccount = accountRepository.save(testAccount);
+        testAccountId = savedAccount.getAccountIdentifier().getAccountId();
+    }
+
+    @AfterEach
+    void tearDown() {
+        accountRepository.deleteAll();
+    }
+
+    @Test
```

Comment:
can you explain how does this work.

Reply:
This makes it so that everything a test runs its it deletes the context of the test. So that it doest produce unnescessary data.